packages/react-debug-tools/src/__tests__/ReactHooksInspectionIntegration-test.internal.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
+'use strict';
+
+let React;
+let ReactTestRenderer;
+let ReactDebugTools;
+
+describe('ReactHooksInspectionIntergration', () => {
```

Comment:
add a custom hook integration test?

Reply:
Would be nice to have some basic testing of the stack parsing+intersection logic too.